src/Akka.Streams.Kafka/Stages/Consumers/Exceptions/StoppingException.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+
+namespace Akka.Streams.Kafka.Stages.Consumers.Exceptions
+{
+    /// <summary>
+    /// Thrown in response to commit/message request commands by consuming actor when in stopping state
+    /// </summary>
+    public class StoppingException : Exception
```

Comment:
Any reason why we shouldn't mark this class as `sealed` ?

Reply:
Yep, you are right.